wildfly/unifiedpush-wildfly-dev/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+# launch wildfly
+exec /opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0 $@
```

Comment:
yeah, we do that in the docker file :-)

https://github.com/aerogear/dockerfiles/blob/master/wildfly/Dockerfile#L22-L23

we copy our -full config to the normal format ;-)